<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Личный календарь</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>201</width>
      <height>341</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="lstTasks">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>421</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Дела</string>
    </property>
    <addaction name="actNew"/>
    <addaction name="actEdit"/>
    <addaction name="actDelete"/>
    <addaction name="separator"/>
    <addaction name="actExit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actNew">
   <property name="text">
    <string>Новое</string>
   </property>
  </action>
  <action name="actEdit">
   <property name="text">
    <string>Редактировать</string>
   </property>
  </action>
  <action name="actDelete">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="actExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
